docs/docs/getting-started/android-setup.md

```diff
@@ -1 +1,236 @@
-# Android Setup
\ No newline at end of file
+# Android Setup
+
+After you've completed the [Rust setup](docs/getting-started/rust-setup.md) you should be able to run `cargo run --bin android`. This will create a new folder called `MoproAndroidBindings`. It should look like the structure
+
+```sh
+MoproAndroidBindings
+├── jniLibs
+│   ├── arm64-v8a
+│   │   └── libuniffi_mopro.so
+│   ├── armeabi-v7a
+│   │   └── libuniffi_mopro.so
+│   ├── x86
+│   │   └── libuniffi_mopro.so
+│   └── x86_64
+│       └── libuniffi_mopro.so
+└── uniffi
+    └── mopro
+        └── mopro.kt
+```
+
+First we will create an android app through Android Studio. If you already have an app project, you can skip this step. We'll do File -> New -> New Project and create an Empty Activity.
+
+![create an android app project](/img/android-example-1.png)
+
+Your android project should be opened now.
+
+Then add jna to `app/build.gradle.kts`
+
+```kts
+dependencies {
+   ...
+   implementation("net.java.dev.jna:jna:5.13.0@aar")
+   ...
+}
+```
+
+![add jna dependency](/img/android-example-2.png)
+
+Sync gradle with `shift`+`cmd`+`O`, or press
```

Comment:
This key combo doesn't work for me. Android studio says this is the correct combo though 🙄

Reply:
I replace the instruction with **File -> Sync Project with Gradle Files**